### PR TITLE
Add greensward skin by Whis

### PR DESCRIPTION
**Pull request by ChillerDragon**
greensward_whis

Using this skin since 6 years. And can highly recommend it.
Created by Whis and part of the DDNet 0.6 client. Whis gave me full permission to contribute this skin to the official client.

### CODE REVIEW

**Comment by fokkonaut** [comment]
eyes x_ninja?

**Comment by fokkonaut** [comment]
Doesnt build? xD

**Comment by ChillerDragon** [comment]
Good catch @fokkonaut fixed.

File: datasrc/skins/greensward.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
+{"skin": {
```
Comment (Ryozuki) [inline datasrc/skins/greensward.json]: is this a new json style standard?

Comment (ChillerDragon) [reply datasrc/skins/greensward.json]: Ye seems like...
also the }} at the end.

**Comment by oy** [comment]
Could you provide the source file (svg)?
You could also do it in private by contacting one of the team via email or irc, in case you don't want to make it public here.

**Comment by Dune-jr** [comment]
The IRC log on that:

> \<Dune> @ChillerDragon i think we can't really add skin parts in the middle of 0.7
> \<Dune> If people use it they won't be seen by everyone
> \<Dune> That defeats the point of official skins
> \<ChillerDragon> hm
> \<ChillerDragon> imo its still early 0.7
> \<ChillerDragon> and it doesnt break too much
> \<ChillerDragon> seing a default skin in a outdated version is fine imo

**Comment by oy** [comment]
Not sure about that. Displaying the standard part for older clients isn't that bad.

**Comment by ChillerDragon** [comment]
@oy 
[greensward.zip](https://github.com/teeworlds/teeworlds/files/3294053/greensward.zip)

**Comment by ChillerDragon** [comment]
https://github.com/teeworlds/teeworlds/issues/2157
Should probably be implemented first before merging this.

File: datasrc/skins/greensward.json (hunk 1 above)
Comment (AssassinTee) [reply datasrc/skins/greensward.json]: side node: also "value":"true" instead of "value":true

Comment (ChillerDragon) [reply datasrc/skins/greensward.json]: @AssassinTee yes sure I fully agree also with @Ryozuki. But this is nothing that belongs in this pr. Consistent style is better than mixed style. We could open a new pr refactoring all skins and discuss it there if we even want to do that.

**Comment by fokkonaut** [comment]
I think we can go without the implementation of #2157, that would take forever. Also, this wouldnt be the only thing looking a bit ugly, we also have twinbop with ears :p

**Comment by Dune-jr** [comment]
I think we should add an issue for a batch update of skins. Get @Sonix- involved, he made a lot of great skins.

**Comment by oy** [comment]
That sounds good 👍

**Comment by fokkonaut** [comment]
We shouldnt wait until we have that one feature. Just merge it, some marking also dont work perfectly right now with ears or other bodies.

**Comment by Dune-jr** [comment]
It can wait for https://github.com/teeworlds/teeworlds/issues/2193. There may be tweaks / standardization efforts, and a pass to select which skin parts we keep. I heard @Zatline is actively working on it :)

**Comment by oy** [comment]
fixed by #2315

**Pull request closed without merging**